src/main/java/org/folio/marccat/resources/domain/FixedField.java

```diff
@@ -448,10 +465,48 @@
   @JsonProperty("formatCode")
   private String formatCode;
 
+  @JsonProperty("subjectDescriptor")
```

Comment:
Aren't these annotations superfluous if you use the same member name value? It has been done?

Reply:
It is solved in other PR

Reply:
It is solved in other PR